interactions.tsv

```diff
@@ -1380,7 +1380,7 @@ sourceOccurrenceId	sourceTaxonId	sourceTaxonName	sourceBodyPartId	sourceBodyPart
 		Epithemia turgida					http://purl.obolibrary.org/obo/RO_0002453	host of			EtSB 										10.1073/pnas.1405222111	(Nakayama et al. 2014)
 		Asterionella formosa					http://purl.obolibrary.org/obo/RO_0002445	has parasite			Chytrids										10.1038/ismej.2013.73	(Gsell et al. 2013)
 		Rhizosolenia hebetata					http://purl.obolibrary.org/obo/RO_0002453	host of			Richelia intracellularis										WOS:000325596800006	(Madhu et al. 2013)
-		Pseudo nitzschia 					http://purl.obolibrary.org/obo/GO_0085030	mutualism			Bacteria										10.1038/ismej.2013.138	(Sison-Mangus et al.2014)
+		Pseudo nitzschia 					http://purl.obolibrary.org/obo/RO_0002442	mutualism			Bacteria										10.1038/ismej.2013.138	(Sison-Mangus et al.2014)
```

Comment:
@debpaul Thanks for using http://purl.obolibrary.org/obo/RO_0002442 instead. Your usage actually uncovered that GloBI didn't support the mutualist interaction type (e.g., from travis log: ```found unsupported interactionTypeId [http://purl.obolibrary.org/obo/RO_0002442]```). I've added support just now via https://github.com/jhpoelen/eol-globi-data/commit/da3ac13472795e93ee55dc74d56fc3e43deac73d . So, on the next GloBI release (within a couple of days), the mutualist interaction type will be available. In short: you helped improve not only this dataset, but GloBI itself!